__tests__/dotnet.test.ts

```diff
@@ -6,6 +6,20 @@ import { Platform } from '../src/platform';
 import { ToolsDirectory } from '../src/toolsDirectory';
 
 const targetDirectory = path.join('target', 'directory');
+const cakeVersion = '0.35.0';
+const installedCakeVersion = '0.34.0';
+const containsFileWithOldToolInstalled = (path: string) => {
+  switch(path)
+  {
+    case 'dotnet-cake':
+    case 'dotnet-cake.exe':
+    case `.store/cake.tool/${installedCakeVersion}/project.assets.json`:
+    case `.store\\cake.tool\\${installedCakeVersion}\\project.assets.json`:
```

Comment:
Could we replace these two with a single `path.join`?

```javascript
    case path.join('.store', 'cake.tool', installedCakeVersion, 'project.assets.json'):
```

Reply:
addressed.